CMakeLists.txt

```diff
@@ -181,10 +181,16 @@ ELSE() # use it from the install folder where is properly installed for MacOS
 ENDIF()
 set(OMTLMSIMULATOR_3RDPARTY_INCLUDEDIR ${PROJECT_SOURCE_DIR}/OMTLMSimulator/3rdParty/misc/include)
 IF(WIN32)
-  set(OMTLM_LINKFLAGS omtlmsimulator wsock32 ws2_32)
-ELSE()
+  IF (MINGW)
+    # adrpo: link with the full path to the dll on MINGW
```

Comment:
That was exactly what I did but couldn't get it into the CMakeLists how I wanted it. :+1: